src/main/java/org/jitsi/jicofo/xmpp/BaseBrewery.java

```diff
@@ -0,0 +1,380 @@
+/*
+ * Jicofo, the Jitsi Conference Focus.
+ *
+ * Copyright @ 2015 Atlassian Pty Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jitsi.jicofo.xmpp;
+
+import net.java.sip.communicator.impl.protocol.jabber.extensions.*;
+import net.java.sip.communicator.service.protocol.*;
+import net.java.sip.communicator.service.protocol.event.*;
+import net.java.sip.communicator.util.*;
+import org.jitsi.assertions.*;
+import org.jitsi.jicofo.*;
+import org.jitsi.protocol.xmpp.*;
+import org.jivesoftware.smack.packet.*;
+
+import java.util.*;
+import java.util.concurrent.*;
+
+/**
+ * <tt>BaseBrewery</tt> manages the pool of service instances which
+ * exist in the current session. Does that by joining "brewery" room where
+ * instances connect to and publish their's status in MUC presence.
+ * <tt>AbstractPacketExtension</tt> is the packet extension that will be used
```

Comment:
Yep, it used to be AbstractPacketExtension and at one moment I chang it to PacketExtension and apparently I haven't updated the docs.